Python/chapter02/2.4 - Partition/miguel_2.4_soln.py

```diff
@@ -92,7 +92,7 @@ def pop_head(self) -> Optional[Node]:
 
     def append(self, ll: object):
         if not isinstance(ll, LinkedList):
-            return TypeError
+            raise TypeError(...)
```

Comment:
err, i didn't mean a literal `...` ^^;; you can use `raise IndexError('pop from empty list')` just like `[].pop()`.